### DISCUSSION TITLE
Where did the Sound_Card_WM8960_Hi Fi_HAT Wiki page go?

### DISCUSSION THREAD

**Discussion by jorgemiar** [MiczFlor/RPi-Jukebox-RFID#1355] [Q&A]
Hi, I built a phoniebox a month ago and remember checking the [Sound_Card_WM8960_Hi Fi_HAT](https://github.com/MiczFlor/RPi-Jukebox-RFID/wiki/Sound_Card_WM8960_Hi-Fi_HAT) wiki page as an installation guide. I went to check on it today but it is empty/it shows "Create new page". What happened to it? Is there any way of checking the page history? If someone creates a new empty page will we be able to see the history?

**Comment by s-martin** [comment]
@MiczFlor can you see any deleted wiki page?

**Comment by oberhamsi** [comment]
i was looking for the same thing and it seems the content was changed because the original drivers are no longer supported. see the changes here: https://github.com/MiczFlor/RPi-Jukebox-RFID/wiki/Sound_Card_WM8960_Hi-Fi_HAT/_compare/456fe0962b6020cb35273706ff6ca600c29e9122...41a98afdb190ea8a343a6402f75999a7ff281bd7
